imx/interface.go

```diff
@@ -0,0 +1,107 @@
+package imx
+
+import (
+	"context"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/immutable/imx-core-sdk-golang/imx/api"
+	"github.com/immutable/imx-core-sdk-golang/imx/internal/contracts"
+)
+
+var (
+	Sandbox = Environment{
+		BaseAPIPath:                 "https://api.ropsten.x.immutable.com",
+		EthereumRPC:                 "https://eth-ropsten.alchemyapi.io/v2/",
+		RegistrationContractAddress: "0x6C21EC8DE44AE44D0992ec3e2d9f1aBb6207D864",
+		CoreContractAddress:         "0x4527BE8f31E2ebFbEF4fCADDb5a17447B27d2aef",
+		ChainID:                     big.NewInt(3), // Ropsten
+	}
+	Mainnet = Environment{
+		BaseAPIPath:                 "https://api.x.immutable.com",
+		EthereumRPC:                 "https://eth-mainnet.alchemyapi.io/v2/",
+		RegistrationContractAddress: "0x72a06bf2a1CE5e39cBA06c0CAb824960B587d64c",
+		CoreContractAddress:         "0x5FDCCA53617f4d2b9134B29090C87D01058e27e9",
+		ChainID:                     big.NewInt(1),
+	}
+)
+
+// Config is used to initialise NewClient object.
+type Config struct {
+	AlchemyAPIKey string
+	APIConfig     *api.Configuration
+	Environment
+}
+
+// Environment holds ethereum network and contract address information. Part of the config.
```

Comment:
Is this partial sentence a typo?
```suggestion
// Environment holds Ethereum network and contract address information.
```